charts/home-assistant/0.0.1/Chart.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v2
+appVersion: 2020.12.1
```

Comment:
current release

```suggestion
appVersion: 2021.2.2
```

Version `2021.2.0` adds Z-Wave JS integration, which uses Z-Wave JS Server 
- Zwave JS Server is provided with ZwaveJS2MQTT

At minimum, version `2021.1.5` is required to address these security vulnerabilities

- https://www.home-assistant.io/blog/2021/01/23/security-disclosure2/

Reply:
Thanks for the headsup!

Also, as you added the zwave dongles, this is quite relevant indeed!